### ISSUE TITLE
Collect minimal sanity telemetry for Alpha

### ISSUE THREAD

**Issue by Dexterp37** [mozilla-rally/rally-core-addon#371]
In order to assess the health of Alpha, we need at least these data points:

- the version of the core-addon (at enrollment);
- the version of the study (at enrollment);

The above would allow us to tie enrollment to specific addon versions.

@rhelmer @hamilton thoughts?

**Comment by Dexterp37** [comment]
This is the related pipeline bug to allow decoding of the payload: https://bugzilla.mozilla.org/show_bug.cgi?id=1691807

**Comment by hamilton** [comment]
cc @changecourse – would this require an update to the privacy notice language? Here is what it is currently:

```
Your Interaction Data
______________________
This includes your <b>interactions with Firefox</b>, like the number and type of Firefox Add-ons you have installed, 
your interactions with the Rally add-on itself, or the duration of your active browsing session.

```

**Comment by marniepw** [comment]
@litkaemily FYI

**Comment by litkaemily** [comment]
This is fine. We need to add these data elements to the SUMO page where it lists all of the data that we collect (or wherever that schema is living). Also, this would "technical data" and not "interaction data."

**Comment by hamilton** [comment]
Got it, was a bit confusing since the technical data section says this:

```
Your Device's Technical Data
-----------------------------

This includes your device's <b>operating system</b> and <b>IP address</b>
(which we'll only hang on to temporarily).
```

@litkaemily @changecourse does anything need to change in the technical data part of the privacy policy to accommodate this?

**Comment by litkaemily** [comment]
If we can make changes, the following would be good to be comprehensive: This includes your device's operating system, add-on and study versions, and IP address (which we'll only hang on to temporarily).

But lmk if it's too late to make changes

**Comment by Dexterp37** [comment]
> 
> 
> If we can make changes, the following would be good to be comprehensive: This includes your device's operating system, add-on and study versions, and IP address (which we'll only hang on to temporarily).

I'm not sure I understand here, so please bear with me. Does this mean that, every time we add a new data point (e.g. technical) we should change the privacy policy there?

@litkaemily can't this point to the [data documentation](https://github.com/mozilla-rally/rally-core-addon/blob/master/docs/DATA_COLLECTION.md) (or some artefact derived from it) so that we can leverage the data-review process?

**Comment by litkaemily** [comment]
No, every time we add a new data point we do not need to update the privacy notice (so long as it's not materially new). But the current notice doesn't link out to the full schema because we determined that it was in fact comprehensive; so, no need to give redundant information. 

@changecourse can we just include the "read more here" link in the notice so that we can update the scheme instead of the notice each time?

**Comment by marniepw** [comment]
Ask @rhelmer if the AMO stats page covers the requirements for this ticket.

**Comment by rhelmer** [comment]
> Ask @rhelmer if the AMO stats page covers the requirements for this ticket.

I don't think they do, I expect we'll want to treat AMO analytics as supplementary/operational more than anything. They are all aggregate, can't really dig in too much. Definitely can't get interaction data or anything like that, it's more like DAU, current versions, attribution codes, etc.
